shell/browser/osr/osr_render_widget_host_view.cc

```diff
@@ -280,9 +281,9 @@ OffScreenRenderWidgetHostView::~OffScreenRenderWidgetHostView() {
         content::DelegatedFrameHost::HiddenCause::kOther);
   delegated_frame_host_->DetachFromCompositor();
 
-  delegated_frame_host_.reset(NULL);
-  compositor_.reset(NULL);
-  root_layer_.reset(NULL);
+  delegated_frame_host_.reset(nullptr);
```

Comment:
Isn’t `nullptr` the default value for `reset()`?

Reply:
Yes, it is -- this check was just changing `NULL` to `nullptr`.

Updated manually in e7322095e0fd917f7f65d9d52a8103d561d2d373